cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js

```diff
@@ -149,20 +149,20 @@ function getPropertiesForPlan(plan, currencyCode) {
 /**
  * Returns if the current payment option is pnx 2,3 or 4
  * @param  {Object} plan any alma plan
- * @returns {boolean} true means we can use fragment
+ * @returns {boolean} true means we can use inpage
  */
 function isPnx(plan) {
-    return plan.installments_count <= 4;
+    return plan.installments_count <= 4 && plan.deferred_days <= 0;
```

Comment:
isPnx function is now with Pay Now, Do you not separate PnX and Pay Now ?

Reply:
I think it's function name is not good

Reply:
Done